tests/APITests/APITest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Test\APITests;
+
+use App\Database\ConnectionManager;
+use App\Database\QueryBuilder;
+use App\Database\QueryWriterFactory;
+use App\Mapper\WordMapper;
+use App\Model\Word;
+use App\Repository\WordRepository;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+
+class APITest extends TestCase
+{
+    private static \PDO $connection;
+
+    public static function setUpBeforeClass(): void
+    {
+        self::$connection = ConnectionManager::getConnection();
+    }
+
+    public function tearDown(): void
+    {
+        $query = 'SET FOREIGN_KEY_CHECKS=0;'
+            . 'TRUNCATE TABLE words;'
+            . 'SET FOREIGN_KEY_CHECKS=1;';
+        self::$connection->exec($query);
+    }
+
+    public static function wordProvider(): array
+    {
+        return [
+            [
+                [
```

Comment:
Add names to test cases to indicate what they are testing.

First tip in: https://blog.martinhujer.cz/how-to-use-data-providers-in-phpunit/